api/nakadi-event-bus-api.yaml

```diff
@@ -1024,6 +1024,26 @@ definitions:
         description: |
           Provides ability to set internal Nakadi parameters.
 
+      write_scopes:
+        type: array
+        items:
+          type: string
+        description: |
+          Provides ability to set oauth2 scopes to be able to publish an Event. The event publisher is cheked for
```

Comment:
I think the idea is clever, but what about our manual http://zalando.github.io/nakadi-manual/? I would say that the API definition should merely describe the semantics of fields and the manual should have a more extensive usage description.